### PR TITLE
HELP BUTTON modal made but button overlaps when scrolling down

### PR DESCRIPTION
**Pull request by tinysheep007**
#151 more styling needed

### CODE REVIEW

**Comment by lucentfong** [comment]
Hi @tinysheep007  sorry for the delay - we will have your code reviewed sometime before the end of this week

**Comment by GabrielWeinbrenner** [comment]
Hi @tinysheep007. Please use https://mui.com/material-ui/react-modal/. You do not have to create a modal for this one

File: challenge/src/components/Pages/Home.jsx [hunk 1]
```diff
@@ -12,37 +12,46 @@ import Mando from "../Mando";
...
-                <PartyParrot/>
-                <ElonMusk />
-            </div>
+  const [questionOpen, setquestionOpen] = useState(false);
+
+
+  return (
+    <div>
+      <div>
+
```
Comment (GabrielWeinbrenner) [inline challenge/src/components/Pages/Home.jsx:23]: Use the modal on material ui instead

**Review by GabrielWeinbrenner — CHANGES_REQUESTED** [review]

**Comment by tinysheep007** [comment]
hey man i commited again

**Pull request closed without merging**